x/stake/handler.go

```diff
@@ -17,18 +16,26 @@ func NewHandler(k keeper.Keeper, govKeeper gov.Keeper) sdk.Handler {
 		// NOTE msg already has validate basic run
 		switch msg := msg.(type) {
 		case types.MsgCreateValidatorProposal:
+			if sdk.IsUpgrade(sdk.BEPHHH) {
+				return sdk.ErrMsgNotSupported("MsgCreateValidatorProposal disabled in BEP-HHH").Result()
+			}
 			return handleMsgCreateValidatorAfterProposal(ctx, msg, k, govKeeper)
 		case types.MsgRemoveValidator:
+			if sdk.IsUpgrade(sdk.BEPHHH) {
+				return sdk.ErrMsgNotSupported("MsgRemoveValidator disabled in BEP-HHH").Result()
+			}
 			return handleMsgRemoveValidatorAfterProposal(ctx, msg, k, govKeeper)
-		// disabled other msg handling
-		//case types.MsgEditValidator:
-		//	return handleMsgEditValidator(ctx, msg, k)
-		//case types.MsgDelegate:
-		//	return handleMsgDelegate(ctx, msg, k)
-		//case types.MsgBeginRedelegate:
-		//	return handleMsgBeginRedelegate(ctx, msg, k)
-		//case types.MsgBeginUnbonding:
-		//	return handleMsgBeginUnbonding(ctx, msg, k)
+		// Beacon Chain New Staking in BEP-HHH
+		case types.MsgCreateValidator:
+			return handleMsgCreateValidator(ctx, msg, k)
+		case types.MsgEditValidator:
```

Comment:
For `MsgEditValidator`, can we make it to support edit `PubKey`, we get some requirement from validator maintainer that they want to change consensus key.

Reply:
Sure. Do we need to add it for BSC either?

Reply:
BC MsgEditValidator  added

Reply:
Yes, BSC want this feature too.